app/controllers/application_controller.rb

```diff
@@ -2488,7 +2488,7 @@ def find_filtered(db, count, options = {})
     if db.respond_to?(:find_filtered) && !mfilters.empty?
       result = db.find_tags_by_grouping(mfilters, :conditions => options[:conditions], :ns => "*")
     else
-      result = db.find(count, options)
+      result = db.where(options)
```

Comment:
is this really `where(options)` ?
I think it should be `apply_legacy_finder_options(options)`